### DISCUSSION TITLE
refresh app.input setting

### DISCUSSION THREAD

**Discussion by rwst** [hydraulic-software/conveyor#78] [Q&A] [answered]
I just succesfully did `make site` for a Kotlin multiplatform project. Many thanks for your work on Conveyor! The immediate question I have: how do I refresh the `app.inputs` setting after a change to dependencies in `build.gradle.kts`?

**Comment by mikehearn** [comment, marked as answer]
If you're using the default approach with a hashbang include (`include "#!./gradlew -q printConveyorConfig`) then you don't need to do anything. It should pick up the change automatically.

**Comment by rwst** [comment]
Confirmed, thanks.
